src/migrations/20220922030802-update-contact-enabled.ts

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  async up(db) {
+    try {
+      const collection = db.collection("users")
+      const result = await collection.updateMany(
+        { role: { $ne: "user" } },
```

Comment:
maybe editor could be excluded also.

Reply:
Fixed https://github.com/GaloyMoney/galoy/pull/1696/commits/ab3bd473794af580a5739e38e2862bf15a03b806